### DISCUSSION TITLE
'screentime' mod suggestion

### DISCUSSION THREAD

**Discussion by Axeanz** [ppy/osu#13110] [Ideas]
to keep it short (because I don't know what else to say about this suggestion):
how about giving you a bigger punishment/challenge when you miss by making the hitcircle you miss stay on the screen with a red tint for **OD * (HP/3.2)** seconds (don't use this formula btw)
OD=7 HP=5, so the circles you miss stay on the screen for 10.9 seconds.
this is purely meant for using on easier/low misscount maps, that's why they should stay on the screen for such a 'long' time.

it will act like a normal hitcirle for mods such as FL, blinds and traceable.
with HD, it will act like a normal hitcircle, but when missed will stay visible for *x* seconds
Mod name: Screentime
Mod description: Take a good look at your mistakes

**Comment by peppy** [comment]
this needs to be renamed or closed
